webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetailsFile.scala

```diff
@@ -313,9 +313,9 @@ final class FileUserDetailListProvider(roleApiMapping: RoleApiMapping, authorisa
   /**
    * Reload the list of users. Only update the cache if there is no errors.
    */
-  def reloadPure(): IOResult[Unit] = {
+  def reloadPure(resetRoles: Boolean): IOResult[Unit] = {
     for {
-      config <- UserFileProcessing.parseUsers(roleApiMapping, file, authorisationLevel.userAuthEnabled, reload = true)
+      config <- UserFileProcessing.parseUsers(roleApiMapping, file, authorisationLevel.userAuthEnabled, resetRoles)
```

Comment:
this seems to be the opposite of what was done in https://github.com/Normation/rudder/pull/4661
Can you double check that we are not undoing something that was done willingnesly ?